src/client/components/ModelViewRegisteringForAnEvent/ModelViewRegisteringForAnEvent.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { AppHeader } from '../Appheader/AppHeader.component';
+import InputLink from '../Input/Input';
+import Dropdown from '../Dropdown/DropDown';
+import { Button } from '../Button/Button';
+import { Card } from '../Card/Card';
+import Footer from '../footer/footer';
+import Logo from './HYF.logo.svg';
+import Modal from '../Modal/Modal';
+import './ModelViewRegisteringForAnEvent.css';
+import { text } from '@storybook/addon-knobs';
+
+const questions = [
+  {
+    id: '01',
+    value: 'Add Question here',
+  },
+  {
+    id: '02',
+    value: 'question2',
+  },
+  {
+    id: '03',
+    value: 'question3',
+  },
+];
+
+const ModelViewRegisteringForAnEvent = () => {
+  const [value, setValue] = useState('');
+
+  const submit = () => {
+    return 'submit';
```

Comment:
No need to return anything here. Instead, you need to call `props.onSubmit` with the values `topic` and whatever you got from `Input`.
This way the consumer of `Modal` will get the information submitted by the user.